modules/402-ingress-nginx/hooks/minimal_ingress_nginx_version.go

```diff
@@ -80,6 +81,9 @@ func discoverMinimalNginxVersion(input *go_hook.HookInput) error {
 		ctrl := s.(ingressNginxController)
 		if ctrl.Version == "" {
 			ctrl.Version = input.Values.Get("ingressNginx.defaultControllerVersion").String()
+			if ctrl.Version == "0.33" {
+				isDisruptionUpdate = true
+			}
```

Comment:
We don't we migrate other versions? Like 0.40+.

Reply:
Because it's not about deprecating all old controllers. It's a preparation to change only one value [here](https://github.com/deckhouse/deckhouse/blob/main/modules/402-ingress-nginx/openapi/config-values.yaml#L4)

Reply:
Got it.